{{ cookiecutter.repo_name }}/{{ cookiecutter.pkg_name }}/config/settings/common.py

```diff
@@ -135,15 +134,30 @@ class Common(Configuration):
         'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
         'django.contrib.messages.middleware.MessageMiddleware',
         'django.middleware.clickjacking.XFrameOptionsMiddleware',
+        'django.middleware.security.SecurityMiddleware',
     )
 
     ROOT_URLCONF = 'config.urls'
 
     WSGI_APPLICATION = 'config.wsgi.application'
 
-    TEMPLATE_DIRS = (
-        os.path.join(BaseDir.BASE_DIR, 'templates'),
-    )
+    TEMPLATES = [
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'DIRS': [os.path.join(BaseDir.BASE_DIR, 'templates'), ],
+            'APP_DIRS': True,
+            'OPTIONS': {
+                'context_processors': [
+                    'django.template.context_processors.debug',
+                    'django.template.context_processors.request',
+                    'django.contrib.auth.context_processors.auth',
+                    'django.contrib.messages.context_processors.messages',
+                    'config.context_processors.django_version',
+                ],
+                'debug': values.Value('DEBUG', environ_name='TEMPLATE_DEBUG'),
```

Comment:
Thats much better now! Also, if people want to change it later, they see how to do it easly (e.g. if they use multiple template engines).